build.cake

```diff
@@ -2,243 +2,118 @@
 // TOOLS
 //////////////////////////////////////////////////////////////////////
 #tool "nuget:?package=GitVersion.CommandLine&version=4.0.0-beta0007"
-#addin "MagicChunks"
+#tool "nuget:?package=NUnit.ConsoleRunner"&version=3.6.1

```

Comment:
@droyad given that we have upgraded to xunit, I don't think we need this anymore